src/components/Game/Slots/Slots.js

```diff
@@ -3,29 +3,45 @@ import PropTypes from 'prop-types';
 
 import classes from './Slots.scss';
 
+import { WORD_LENGTH } from '../../../constants/constants';
+import { generateID } from '../../../helpers';
+
+
 const Slots = ({ lettersInSlots, clicked }) => {
-    const slotsNodes = lettersInSlots.map((letter, index) => (
-        <li
-            key={letter.id}
-            className={classes.Slot}
-            onClick={() => clicked(letter.id, index)}
-        >
-            {letter.value}
-        </li>
-    ));
+    const slotsNodes = [];
+    let slotIndex = 0;
+
+    while (slotIndex < WORD_LENGTH) {
+        const letter = lettersInSlots && lettersInSlots[slotIndex];
+        const letterIndex = slotIndex;
+
+        slotsNodes.push(
```

Comment:
this is what I would rather do:
```
<ul className={classes.Slots}>
  {[...Array(WORD_LENGTH).keys()].map(i => (
    <li>…</li>
  ))}
</ul>
```
that way you don't need `slotsNodes`, `slotIndex`